org.opent2t.sample.hub.superpopular/com.insteon.hub/js/manifest.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <schemas>
+    <schema id="org.opent2t.sample.hub.superpopular" main="true" type="OCF" />
+    <schema id="oic.core" type="OCF" />
+    <schema id="oic.baseResource" type="OCF" />
+  </schemas>
+  <!-- associated onboarding module -->
+  <onboarding id="org.opent2t.onboarding.Insteon">
```

Comment:
Fixed.

---
In reply to: [82258512](https://github.com/openT2T/translators/pull/107#discussion_r82258512) [](ancestors = 82258512)